### PR TITLE
Added jest and couple of unit tests for DDW-706

### PR DESCRIPTION
**Pull request by danielmain**
This PR CHANGES.

Add Jest library and Unit Tests to the new added functions introduced in branch DDW-706

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Review by DominikGuzei — CHANGES_REQUESTED** [review]

File: __tests__/wallet-utils.specs.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
+import BigNumber from 'bignumber.js';
+import { isWalletRewardsWithdrawalPossible, shouldShowEmptyWalletWarning } from '../source/renderer/app/utils/walletUtils';
+
+// As a user using a shelly wallet when sending ADA and the balance after the transactions remains < 10 ADA, the following Warning message is displayed in the send confirmation dialog
```
Comment (DominikGuzei) [inline __tests__/wallet-utils.specs.js:4]: Let's avoid too long lines like this

File: __tests__/assets.specs.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  uniqueId: '789ef8ae89617f34c07f7f6a12e4d65146f958c0bc15a97b4ff169f1',
+}];
+
+describe('Asset helper functions', () => {
+  it('Function hasTokenLeftAfterTransaction returns true in case after the transaction no tokes are left in the wallet', () => {
+    const assetAmounts = ["3","1","1"];
+    const result = hasTokenLeftAfterTransaction(allAssets, assetAmounts);
+    expect(result).toBe(true);
+  });
+  it('Function hasTokenLeftAfterTransaction returns false in case after the transaction tokes are left in the wallet', () => {
```
Comment (DominikGuzei) [inline __tests__/assets.specs.js:50]: Typo in word "tokes" -> should be "tokens"

**Comment by DominikGuzei** [comment]
@danielmain do you know why this was closed? :D

**Comment by danielmain** [comment]
@DominikGuzei long story short, I did not know merging the parent branch would close automatically this PR. Let me recreate it now. By the way, your suggestions are in the code now :)

**Pull request closed without merging**